chart/crds/openshift.yaml

```diff
@@ -0,0 +1,186 @@
+{{- if .Values.openshift.enabled -}}
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
don't have any idea about how helm works
but is there any way we can refer this form the CRD yaml?

instead of adding CRD definition here, do we have a way to import the yaml?
so in future any changes there will automatically be added in charts 😃

Reply:
We should definitely have a way to keep the CRDs in-sync!
We cannot reference anything "outside" the Helm chart, so we'll need to copy the CRDs inside this directory - this should not be difficult to do as part of the release process, though I think it makes sense to handle this in a separate PR :-)

Reply:
Yes, we need to automate that, but it could be done in a follow-up 😉